### ISSUE TITLE
Exception occurred in cucumber2jvm/cucumber3jvm versions when running Scenario Outline Examples

### ISSUE THREAD

**Issue by QA2Dev** [allure-framework/allure-java#291]
When Executing **Scenario Outline** Examples in cucumber in the following versions i.e cucumber2jvm/cucumber3jvm,  I'm getting an exception as given below.

After execution first example scenario, i'm getting the below exception script is not able to run the next example 

### Issue Log 

```
Exception in thread "main" java.util.NoSuchElementException: No value present
        at java.util.Optional.get(Optional.java:135)
        at io.qameta.allure.cucumber3jvm.AllureCucumber3Jvm.getExamplesAsParameters(AllureCucumber3Jvm.java:233)
        at io.qameta.allure.cucumber3jvm.AllureCucumber3Jvm.handleTestCaseStarted(AllureCucumber3Jvm.java:119)
        at cucumber.runner.EventBus.send(EventBus.java:28)
        at cucumber.runner.TestCase.run(TestCase.java:37)
        at cucumber.runner.Runner.runPickle(Runner.java:44)
        at cucumber.runtime.Runtime.runFeature(Runtime.java:120)
        at cucumber.runtime.Runtime.run(Runtime.java:106)
        at cucumber.api.cli.Main.run(Main.java:35)
        at cucumber.api.cli.Main.main(Main.java:18)
```

### Dependency Configuration & Plugin

```
buildscript {
    repositories { jcenter() }
    dependencies {
        classpath "io.qameta.allure:allure-gradle:2.5"
    }
}

apply plugin: 'io.qameta.allure'

allure 
{
        aspectjweaver = true
         version = '2.7.0'
        useCucumberJVM 
   {
        version = '2.7.0'
    }
}

task cucumber(type: Test) {
{

            args = ['--glue','Steps',
                    '--plugin', 'io.qameta.allure.cucumber2jvm.AllureCucumber2Jvm',
                    '--tags', tag,
                    "src/test/resources/features"
            ]
}

dependencies 
{
compile group: 'io.qameta.allure', name: 'allure-cucumber2-jvm', version: '2.7.0'
}

testCompile ("io.cucumber:cucumber-java:2.3.1”)
testCompile ("io.cucumber:cucumber-junit:2.3.1")
compile     ("io.cucumber:cucumber-core:2.3.1")
testCompile ("io.cucumber:gherkin:4.1.3”)
```

**Comment by letsrokk** [comment]
@QA2Dev can you please add example of feature file here for reference too? thanks!

**Comment by QA2Dev** [comment]
@regression
Feature: Canvas Program Library

  #Verify Canvas Program Library feature

  Background:

    Given I am on the Program Library page
    Then I see program library page is launched


  Scenario Outline: Visit Program Library page and verify all the programs listed in library


    When I tap on "<program>" listed in program library page
    Then I see canvas program and build section of the "<program>" program

    @motorPower_program
    Examples:

      | program                     |
      | MotorPower                  |

    @increaseSpeed_program
    Examples:

      | program                     |
      | IncreaseSpeed               |

    @motorDirectionAndMotorTime_program
    Examples:

      | program                     |
      | MotorDirectionAndMotorTime  |

**Comment by QA2Dev** [comment]
@letsrokk Please let me know allureCucumber3JVM build version to be added in dependency, for which Fix is given.

dependencies 
{
compile group: 'io.qameta.allure', name: 'allure-cucumber2-jvm', version: '?'
}

**Comment by letsrokk** [comment]
@QA2Dev this bug fix will be released in next version of Allure, don't know when it will be released though. Maybe @baev can help us here

As for dependencies,

```
compile group: 'io.qameta.allure', name: 'allure-cucumber3-jvm', version: 'NEW_VERSION_OF_ALLURE'
```

With Cucumber JVM 3.x you should use `allure-cucumber3-jvm`
With Cucumber JVM 4.x you should use `allure-cucumber4-jvm` (not available yet, will be released with next release of Allure adapters)

**Comment by baev** [comment]
released as 2.9.0 (will be available in maven central within an hour)

**Comment by QA2Dev** [comment]
Thank you
